src/golang/cmd/migrator/versions/000018_add_serialization_value_to_param_op/queries.go

```diff
@@ -0,0 +1,69 @@
+package _000018_add_serialization_value_to_param_op
+
+import (
+	"context"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/utils"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/google/uuid"
+)
+
+var (
+	serilization_type_key = "serialization_type"
+	spec_field            = "spec"
+)
+
+type Operator struct {
+	Id     uuid.UUID `db:"id" json:"id"`
+	OpSpec Spec      `db:"spec" json:"spec"`
+}
+
+func getAllOperators(
+	ctx context.Context,
+	db database.Database,
+) ([]Operator, error) {
+	query := "SELECT id, spec FROM operator;"
+
+	var response []Operator
+	err := db.Query(ctx, &response, query)
+	return response, err
+}
+
+func updateParamOperatorWithNewSpec(
+	ctx context.Context,
+	operator Operator,
+	db database.Database,
+) error {
+	// Before the migration all param were json serializable only so we can assign the type as json to all
```

Comment:
@HarisChoudhary not really. Just because we required the value to be json serializable doesn't mean the underlying value should now correspond to artifact type or serialization type `JSON`. I implemented logic to infer the types in my migration step 16 for artifact results, so maybe you can borrow the logic there? Worst case we might need to trigger some Python jobs to run the `infer_artifact_type` function to figure out the exact type.

Reply:
Got it, thanks for the clarification. I'll adjust this code accordingly